src/components/SortableCommentList.js

```diff
@@ -177,67 +178,65 @@ export class SortableCommentListComponent extends Component {
     const pluginContent = showCommentList && displayVisualization ? this.renderPluginContent() : null;
     return (
       <div>
-        {canComment || showCommentList ?
-          <div className="sortable-comment-list">
-            {commentForm}
-            <div>
-              {showCommentList &&
-              <h2>
-                <FormattedMessage id="comments" />
-                <div className="commenticon">
-                  <Icon name="comment-o" />&nbsp;{get(sectionComments, 'count') ? sectionComments.count : '0'}
-                </div>
-              </h2>}
-              {pluginContent}
-              {showCommentList &&
-                <div className="row">
-                  <form className="sort-selector">
-                    <FormGroup controlId="sort-select">
-                      <ControlLabel>
-                        <FormattedMessage id="commentOrder" />
-                      </ControlLabel>
-                      <div className="select">
-                        <FormControl
-                          componentClass="select"
-                          onChange={event => {
-                            this.fetchComments(section.id, event.target.value);
-                          }}
-                          value={get(sectionComments, 'ordering')}
-                        >
-                          {keys(ORDERING_CRITERIA).map(key =>
-                            <FormattedMessage id={key} key={key}>
-                              {(message) => <option value={ORDERING_CRITERIA[key]}>{message}</option>}
-                            </FormattedMessage>
-                          )}
-                        </FormControl>
-                      </div>
-                    </FormGroup>
-                  </form>
-                </div>}
+        {section.commenting !== 'none' &&
+        <div className="sortable-comment-list">
+          {commentForm}
+          <div>
+            <h2>
+              <FormattedMessage id="comments" />
+              <div className="commenticon">
+                <Icon name="comment-o" />&nbsp;{get(sectionComments, 'count') ? sectionComments.count : '0'}
```

Comment:
Here, we shouldn't use `sectionComments.count` but, rather, `section.n_comments`. The former is zero when fetching, but the latter always contains the corrent count. Very seldom do we need `sectionComments.count` anywhere.